extension/awsmiddleware/README.md

```diff
@@ -3,27 +3,27 @@
 An AWS middleware extension provides request and/or response handlers that can be configured on AWS SDK v1/v2 clients.
 Other components can configure their AWS SDK clients using the `awsmiddleware.ConfigureSDKv1` and `awsmiddleware.ConfigureSDKv2` functions.
```

Comment:
Does this need to be updated as well?
There is no function `awsmiddleware.ConfigureSDKv1`.
It is now `awsmiddleware.NewConfigurer(...).ConfigureSDKv1(...)`

Reply:
Yep. Missed this.